device-connectors/src/testflinger_device_connectors/devices/muxpi/muxpi.py

```diff
@@ -641,17 +641,28 @@ def check_test_image_booted(self):
             )
 
         while time.time() - started < 1200:
-            try:
-                time.sleep(10)
+            time.sleep(10)
 
-                if boot_check_url is not None:
+            if boot_check_url is not None:
+
+                logger.info(
+                    f"Checking {boot_check_url} to confirm boot success ..."
+                )
```

Comment:
Please use f-string notation when using the logging functions

```suggestion
                logger.info(
                    "Checking %s to confirm boot success ...", boot_check_url
                )
```

Reply:
Fixed, resolving.